e2e/tests/transfer/forwarding_test.go

```diff
@@ -33,6 +33,28 @@ func (s *TransferForwardingTestSuite) SetupSuite() {
 // TestForwarding_WithLastChainBeingICS20v1_Succeeds tests the case where a token is forwarded and successfully
 // received on a destination chain that is on ics20-v1 version.
 func (s *TransferForwardingTestSuite) TestForwarding_WithLastChainBeingICS20v1_Succeeds() {
+	channelBToCCallback := func() ibc.ChannelOutput {
+		ctx := context.TODO()
+		// Creating a new path between chain B and chain C with a ICS20-v1 channel
+		opts := s.TransferChannelOptions()
+		opts.Version = transfertypes.V1
+		chains := s.GetAllChains()
+		channelBtoC, _ := s.CreatePath(ctx, chains[1], chains[2], ibc.DefaultClientOpts(), opts)
+		s.Require().Equal(transfertypes.V1, channelBtoC.Version, "the channel version is not ics20-1")
+		return channelBtoC
+	}
+	s.testForwardingThreeChains(channelBToCCallback)
+}
```

Comment:
These two tests are identical, they only differ in the fact that the channel from B to C in one is the regular one we create in the setup, while in this one is a v1 channel.

I extracted all to a common function which accepts a parameter that specifies how the channel from B to C is obtained. This test overrides it, while the other one (below) simply returns the one already created.